spec/features/shelterreviews/shelter_review_creation_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe "when i visit a shelter show page and click new review link" do
+  describe "i am taken to a review form where i can enter a review and optional image" do
```

Comment:
I am fairly sure it is convention to stick to one describe per test block... but I would need confirmation.

Reply:
As far as I know you can have as many as you like.  I had 4 of them in one of my blocks in the project I submitted.